src/support.sch

```diff
@@ -145,6 +172,37 @@
         </rule>
     </pattern>
 
+    <pattern id="graphic">
+        <rule context="graphic[not(starts-with(@xlink:href, 'http://') or starts-with(@xlink:href, 'https://'))]" role="warn">
+            <assert test="true">
+                &lt;<name/>&gt; without an absolute HTTP @xlink:href is ignored.
+            </assert>
+        </rule>
+        <rule context="graphic[not((@mimetype='image' and @mime-subtype='jpeg') or ('.jpg'=substring(@xlink:href, string-length(@xlink:href)-3)))]" role="warn">
+            <assert test="true">
+                &lt;<name/>&gt; is ignored if it is not a JPEG.
+            </assert>
+        </rule>
+        <rule context="graphic">
+            <assert test="count(preceding-sibling::graphic[(@mimetype='image' and @mime-subtype='jpeg') or ('.jpg'=substring(@xlink:href, string-length(@xlink:href)-3))])=0" role="warn">
+                Extra &lt;<name/>&gt; is ignored.
+            </assert>
+        </rule>
+    </pattern>
+
+    <pattern id="graphic_parent">
+        <rule context="graphic[parent::*]">
+            <let name="parent" value="name(..)"/>
+            <assert test="
+                $parent='body'
+                or $parent='fig'
+                or $parent='sec'
+            " role="warn">
+                &lt;<name/>&gt; in &lt;<value-of select="$parent"/>&gt; is ignored.
+            </assert>
+        </rule>
+    </pattern>
```

Comment:
FYI - I think some publishers capture `graphic` in `alternatives` for different formats.

Example from [JATS documentation](https://jats.nlm.nih.gov/archiving/tag-library/1.1d1/n-xbt0.html)
```
... 
<fig id="f3" position="float">
<label>FIG. 3.</label>
<caption>...</caption>
<alternatives>
<graphic specific-use="print" xlink:href="1.4821168.figures.highres.f3.zip"></graphic>
<graphic specific-use="online" xlink:href="1.4821168.figures.online.f3.jpg"></graphic>
</alternatives>
</fig>
...
```

Reply:
For this first pass we'll just be handling a single JPEG (https://github.com/libero/libero/issues/130#issuecomment-479391333); we'll add support for things like `<alternatives>` at some point (though it could be just a rewrite step as part of https://github.com/libero/libero/issues/131).